src/styles/addbook.css

```diff
@@ -0,0 +1,71 @@
+:root {
+  --font-family-primary: 'Montserrat', sans-serif;
+}
+
+form {
+  margin-top: 20px;
+  border-top: 1px solid rgb(214, 214, 214);
+  padding: 10px 0 50px 0;
```

Comment:
### Your Form
![Student-BookStore](https://user-images.githubusercontent.com/93039359/205461050-cd7c0e44-4767-46f5-a688-4fc95802aec7.PNG)

### Template Form
![bookstore](https://user-images.githubusercontent.com/93039359/205461054-4937519f-98ac-47aa-8c9e-ead3e664c51b.png)

- [ ] Your form is not similar as the given template 😨 Kindly make it similar to the given template to make your project look professional 😊